README.md

```diff
@@ -129,14 +129,14 @@ class YourFavoriteModel(BaseEstimator): # Inherits sklearn base classifier
         pass
 
 # Now you can use your model with `cleanlab`. Here's one example:
-from cleanlab.classification import LearningWithNoisyLabels
-lnl = LearningWithNoisyLabels(clf=YourFavoriteModel())
+from cleanlab.classification import CleanLearning
+lnl = CleanLearning(clf=YourFavoriteModel())
```

Comment:
We should probably change the name of the variable everywhere now. `lnl` doesn't really make sense as a variable name anymore.  I propose we always name the sklearn Estimator to be a variable `model` (sometimes it's currently called `clf` but I'd replace with `model` throughout)  and the CleanLearning-wrapped estimator to be a variable `cl_model` (in all examples, tutorials, readme, etc.)

Reply:
how about replacing `lnl` everywhere with `cl`. I like it since it stands for `cleanlab` too

Reply:
fixed here [ef92446](https://github.com/cleanlab/cleanlab/pull/177/commits/ef92446022cb4754d537f8737ec3dc77772b8856)